include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -791,6 +791,60 @@ The data will be distributed between threads as follows:
   let hasCustomAssemblyFormat = 1;
 }
 
+def WmmaEncodingAttr : DistributedEncoding<"WmmaEncoding", "wmma_encoding", [MmaEncodingTrait]> {
```

Comment:
The nvidia layouts should be in an nvidia namespace/dialect -- is there any reason the AMD one can't be in the AMD dialect?

(We're having internal conversations about making layouts more general so you don't need to do this at all, but that's a medium-term refactoring at best.)

Reply:
I agree with Justin however I don't think we have an AMD dialect at the time and it is probably premature to add one for this.